src/teststack/containers/docker.py

```diff
@@ -136,11 +137,16 @@ def image_get(self, tag):
     def run_command(self, container, command, user=None):
         container = self.client.containers.get(container)
         click.echo(click.style(f'Run Command: {command}', fg='green'))
+        terminal = shutil.get_terminal_size()
         exec_id = container.client.api.exec_create(
             container.id,
             command,
             stdin=True,
             tty=True,
+            environment={
+                'COLUMNS': terminal.columns or 80,
+                'LINES': terminal.columns or 20,
```

Comment:
24 to match the shutil's defaults on main?

Reply:
ahh yeah, for some reason i thought it was 20.